### PR TITLE
feat: link unfurling c# template

### PR DESCRIPTION
**Pull request by yukun-dong**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_sprints/taskboard/Compute%20Team/Microsoft%20Teams%20Extensibility/CY23-5.3?workitem=24120674

### CODE REVIEW

**Comment by JerryYangKai** [comment]
Do we need appSettings.Development.json?

File: templates/csharp/link-unfurling/LinkUnfurling/LinkUnfurlingApp.cs.tpl [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        {
+            Title = "Preview Card",
+            Text = query.Url,
+            Images = new List<CardImage> { new CardImage("https://raw.githubusercontent.com/microsoft/botframework-sdk/master/icon.png") },
+        };
+        var previewAttachment = new Attachment()
+        {
+            ContentType = ThumbnailCard.ContentType,
+            Content = previewCard
+
```
Comment (JerryYangKai) [inline templates/csharp/link-unfurling/LinkUnfurling/LinkUnfurlingApp.cs.tpl:66]: An empty line here

**Review by JerryYangKai — APPROVED** [review]

**Review by JerryYangKai — DISMISSED** [review]

File: templates/csharp/link-unfurling/.webappignore [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.deployment
```
Comment (Siglud) [inline templates/csharp/link-unfurling/.webappignore:1]: I think you don't need this file, cause Dotnet project will upload all files in target folder and ignore nothing.

**Comment by Siglud** [comment]
Maybe you can also edit the `.github/CODEOWNERS` file?

**Review by JerryYangKai — APPROVED** [review]

**Comment by yukun-dong** [comment]
> Maybe you can edit the `.github/CODEOWNERS` file too?

I will create another PR to update.

**Pull request merged**